Dockerfile

```diff
@@ -1,46 +1,10 @@
-# Start from the IQ# base image. The definition for this image can be found at
-# https://github.com/microsoft/iqsharp/blob/master/images/iqsharp-base/Dockerfile.
-# As per Binder documentation, we choose to use an SHA sum here instead of a
-# tag.
-FROM mcr.microsoft.com/quantum/iqsharp-base:0.10.1912.501
+# This uses the latest Docker image built from the samples repository,
+# defined by the Dockerfile in Build/images/samples.
+FROM docker.pkg.github.com/microsoft/quantum/samples:latest
 
 # Mark that this Dockerfile is used with the samples repository.
 ENV IQSHARP_HOSTING_ENV=SAMPLES_DOCKERFILE
```

Comment:
I wonder if we should have different values for both, maybe this one should be SAMPLES_HOSTED?

Reply:
Good point, easy enough to change, yeah.